react-native-libraries.json

```diff
@@ -9637,12 +9637,19 @@
   {
     "githubUrl": "https://github.com/fracht/react-native-input-autofocus",
     "npmPkg": "@alcs/react-native-input-autofocus",
-    "examples" :["https://github.com/fracht/react-native-input-autofocus/tree/main/ExampleApp"],
+    "examples": ["https://github.com/fracht/react-native-input-autofocus/tree/main/ExampleApp"],
     "images": [
       "https://github.com/fracht/react-native-input-autofocus/raw/main/docs/assets/androidExample.gif",
       "https://github.com/fracht/react-native-input-autofocus/raw/main/docs/assets/iosExample.gif"
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/numandev1/react-native-keys",
+    "npmPkg": "https://www.npmjs.com/package/react-native-keys",
```

Comment:
Hey @numandev1, thank you for adding new library to the directyory! 👍 

```suggestion
```

CI fails, because `npmPkg` has an invalid value. It should include just the package name, but in case when package name matches GitHub repository it is not needed at all.

Reply:
@Simek fixed